File: Examples/output/cs/unittesting/example2.cs
namespace Cornichon.Accumulator2
{
  using Microsoft.VisualStudio.TestTools.UnitTesting;

  /// <summary>
  /// Gherkin DSL feature
  /// </summary>
  [TestClass]
  public class Feature
  {
    /// <summary>
    /// Gherkin DSL test
    /// </summary>
    [TestMethod]
    public void AddOneOther()
    {
      var scenario = new Scenarios.AddOneOther();
      scenario.GivenAnInitial6();
      scenario.WhenYouAddA5();
      scenario.ThenTheResultIs11();
    }

    /// <summary>
    /// Gherkin DSL test
    /// </summary>
    [TestMethod]
    public void AddTwoOthers()
    {
      var scenario = new Scenarios.AddTwoOthers();
      scenario.GivenAnInitial6();
      scenario.WhenYouAddA8();
      scenario.WhenYouAddA4();
      scenario.ThenTheResultIs18();
    }
  }
}

File: Examples/output/vb/unittesting/example2.vb
Imports Microsoft.VisualStudio.TestTools.UnitTesting

Namespace Cornichon.Accumulator2

  ' <summary>
  ' Gherkin DSL feature
  ' </summary>
  <TestClass>
  Public Class Feature
    ' <summary>
    ' Gherkin DSL test
    ' </summary>
    <TestMethod>
    Public Sub AddOneOther()
      Dim scenario As Scenarios.AddOneOther = New Scenarios.AddOneOther()
      scenario.GivenAnInitial6()
      scenario.WhenYouAddA5()
      scenario.ThenTheResultIs11()
    End Sub

    ' <summary>
    ' Gherkin DSL test
    ' </summary>
    <TestMethod>
    Public Sub AddTwoOthers()
      Dim scenario As Scenarios.AddTwoOthers = New Scenarios.AddTwoOthers()
      scenario.GivenAnInitial6()
      scenario.WhenYouAddA8()
      scenario.WhenYouAddA4()
      scenario.ThenTheResultIs18()
    End Sub
  End Class
End Namespace
